src/execCli.js

```diff
@@ -72,6 +72,7 @@ export async function execCli(argv) {
   try {
     cli.parse(argv, { run: false })
     await cli.runMatchedCommand()
+    process.exit(0)
```

Comment:
```suggestion
    // the InteractiveLogger runs a setInterval so we need to explicitly call process.exit(0) here to avoid hangin
    process.exit(0)
```